RELEASE.md

```diff
@@ -172,6 +172,89 @@ Steps:
 
 ---
 
+## 🍎 LLVM Apple Containers Release Process
+
+> **📋 Status**: Infrequent releases (approximately every 3 years)
+> **Use Case**: Cross-compilation for macOS targets from Linux environments
+
+### When to Release LLVM Containers
+- New Xcode/macOS SDK versions are available
+- Security updates for LLVM toolchain
+- Compatibility requirements for new macOS targets
+- Critical bug fixes in the cross-compilation toolchain
+
+### Prerequisites
+- Access to Buildkite pipeline: https://buildkite.com/elastic/llvm-apple
+- Understanding of LLVM/Clang toolchain versions
+- Knowledge of macOS SDK compatibility requirements
+
+### Step-by-Step Instructions
+
+#### 1. Prepare for Release
+- Verify the current LLVM/Clang version in the containers
```

Comment:
How would one verify these? Where would they look?

Related: would it be worth mentioning that the `OSX_SDK` variable should also be checked, e.g. https://github.com/elastic/golang-crossbuild/blob/main/go/llvm-apple/Dockerfile.tmpl#L55?